enterprise/cmd/executor-queue/internal/queues/codeintel/transform.go

```diff
@@ -67,6 +68,7 @@ func transformRecord(index store.Index, config *Config) (apiclient.Job, error) {
 					"-root", root,
 					"-upload-route", uploadRoute,
 					"-file", outfile,
+					"-associatedIndex", strconv.Itoa(index.ID),
```

Comment:
```suggestion
					"-associated-index-id", strconv.Itoa(index.ID),
```

From [this flag](https://github.com/sourcegraph/src-cli/commit/b7ed8a5d6039f3ff13ed229f0e5cccb924619d07#diff-9d625c61288bbffc0181fb0183d4f0c119d78b0d1a840652b528773142f874ceR75).

Reply:
are there automated tests for the autoindexer?

Reply:
Not currently. We have integration tests but it's explicit uploading. It's a bit of an effort since we'd have to figure out how to spot-deploy our executor infrastructure. The delivery RFC I'm working on might help solve this, though.